### DISCUSSION TITLE
Add support for major/minor release tags for docker containers (Shared Tags)

### DISCUSSION THREAD

**Discussion by 33Fraise33** [grafana/grafana#42879] [General]
Hi,

We are running grafana internally with docker. We are currently still on v7.4.3. Updating this is a manual task I need to remind some people on and it might be forgotten. There are tools like https://github.com/containrrr/watchtower that allow to auto update containers if you use an image that is updated like the `latest` tag. The issue with this is that you might upgrade from v7 to v8 or from v8 to v9 which is a major version change which might introduce breaking changes. 

Would it be possible to also release the docker containers with support for minor/major release tags? such as: 
`grafana/grafana:8` or `grafana:grafana:8.1`. 

A good example are the tags used at python: https://hub.docker.com/_/python

The docker info on this feature: 
https://github.com/docker-library/faq#whats-the-difference-between-shared-and-simple-tags

**Comment by fl0wx** [comment]
What about: https://github.com/grafana/grafana/discussions/47177#discussioncomment-3157311 ?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
